File: sdk/android/library/src/main/java/com/biglabs/solo/signer/library/models/Result.java
package com.biglabs.solo.signer.library.models;

import com.google.gson.annotations.SerializedName;

public class Result {
    @SerializedName("jsonrpc")
    private String jsonRpc;

    @SerializedName("id")
    private int id;

    @SerializedName("result")
    private String result;

    public String getJsonRpc() {
        return jsonRpc;
    }

    public int getId() {
        return id;
    }

    public String getResult() {
        return result;
    }
}

File: sdk/android/library/src/main/java/com/biglabs/solo/signer/library/models/Result.kt
package com.biglabs.solo.signer.library.models

import com.google.gson.annotations.SerializedName

class Result {
    @SerializedName("jsonrpc")
    var jsonRpc: String? = null

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("result")
    var result: String? = null
}